README.md

```diff
@@ -52,115 +56,48 @@ With Go:
-
-✅ **Limit the blast radius and risk of infrastructure changes**
-
-✅ **Significantly faster run times and lower costs**
-
-✅ **Better ownership management and governance**
-
-✅ **Improved productivity and developer experience while reducing complexity**
-
-✅ **Better observability and security**
-
-✅ **Enables multi-step and multi-IaC use cases**
-
-Terramate Stacks can be cloned, nested, compared and orchestrated. You can also generate code in stacks to keep them
-DRY (e.g., generate files such as the Terraform backend configuration for all stacks that manage Terraform resources).
-
-Terramate Stacks are created using a single command.
-
-```sh
-terramate create path/to/new/stack
-```
-
-## What is an IaC orchestrator?
-
-An orchestrator allows you to execute commands in stacks while defining the order of execution, dependencies, and more.
-Terramate CLI comes with a powerful orchestration and execution engine that allows you to easily invoke commands in stacks.
-
-```sh
-terramate run <CMD>
-```
-
-Terramate CLI also comes with a change detection, that seamlessly integrates with git enabling you to execute commands
-in stacks that contain changes only.
-
-E.g., invoke `terraform apply` in all stacks that contain changes:
-
-```sh
-terramate run --changed -- terraform plan
-```
+For other installation methods, please see the [documentation](https://terramate.io/docs/cli/installation).
 
-Another example is to invoke a command in all stacks that have the tags `k8s`and`prd`, and that contain changes.
 
-```sh
-terramate run --changed --tags k8s:prd -- kubectl apply
-```
-
-Terramate CLI is designed and implemented by long-time DevOps, Cloud and Platform Engineering practitioners based on
-previous experience working with large-scale platforms in some of the most sophisticated companies in the world.
-It focuses on significantly improving the developer experience and productivity and integrates with existing tooling
-in a non-intrusive way without major lock-ins and for a broader audience.
-
-If you like Terramate CLI, please give the project a ⭐ and join our [Discord Community](https://discord.gg/CyzcScEPkc).
-
-## What are the most common use cases of Terramate CLI?
-
-📈 **Implement and maintain highly scalable IaC projects**
-
-Stacks unlock faster execution, better collaboration, and lower blast radius risk. With stacks, parallel deployment
-ensures no more pipeline run bottlenecks.
-
-🔗 **GitOps workflows that run in your existing CI/CD platform**
+## Getting Started
 
-Unlock CI/CD workflows to orchestrate your stacks in your existing CI/CD platform, such as GitHub Actions, GitLab and
-Bitbucket Pipelines. Can be used to **preview changes and plans in Pull Requests** or to **detect infrastructure drift**
-with scheduled runs.
+The fastest way to get started with Terramate is our [getting started guide](https://terramate.io/docs/cli/getting-started/).
 
-🏎 **Deploy significantly faster and more often**
+## Why simply complex codebases?
```

Comment:
Love that you cut out the part about stacks. Stacks are an important concept, but they're an implementation detail.